### PR TITLE
Partial fix

### PR DESCRIPTION
**Pull request by ladinesa**

### CODE REVIEW

File: src/nomad_parser_vasp/schema_packages/vasp_package.py [hunk 1]
```diff
@@ -43,7 +36,7 @@
 Simulation.outputs.m_annotations['xml'] = MappingAnnotationModel(path=".calculation")
 
 Program.name.m_annotations['xml'] = MappingAnnotationModel(
-    path="modeling.generator.i[?name == 'program']"
+    path='modeling.generator.i[?"@name"==\'program\'] | [0].__value',
```
Comment (ndaelman-hu) [inline src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: @ladinesa what does the `[0].__value` option cover?

**Review by ndaelman-hu — APPROVED** [review]

File: src/nomad_parser_vasp/schema_packages/vasp_package.py (hunk 1 above)
Comment (ladinesa) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: this is sort of standard i put up in the xml parser for the case where attributes are defined for a quantity so the quantity value itself is given by __value while the attribute keys are prefixed by `@`, you can modify these when you instantiate XMLParser, with kwarg value_key and attribute_prefix

Comment (ladinesa) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: the index [0] since jmespath will return a list when querying a list

Comment (ndaelman-hu) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: Thx for the clarification. Is the `@` consistent with the node definition in JMesPath?

Comment (ladinesa) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: it has nothing to do with it i think, i am not even familiar with the usage of @ in jmespath tbh. we can change the default probably to some other token if @ is used in jmespath

Comment (ndaelman-hu) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: `@` means the current node: https://jmespath.org/specification.html#spec

Comment (ladinesa) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: ah so, we can use it in place of '.' in my convention?nice to know.

Comment (ndaelman-hu) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: > this is sort of standard

If this should always or frequently be used, shouldn't it be moved to the XML parser?

Comment (ladinesa) [reply src/nomad_parser_vasp/schema_packages/vasp_package.py:39]: you can perhaps define this as a variable in parsers common utils but definitely not in nomad.

**Pull request merged**